src/gha_logs/ansi_up@6.0.6.min.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Bundled by jsDelivr using Rollup v2.79.2 and Terser v5.39.0.
+ * Original file: /npm/ansi_up@6.0.6/ansi_up.js
+ */
```

Comment:
What's the license on this? Can we get it copy/pasted into this comment?

Reply:
Oh yeah, forgot about the license. It's MIT license. Included it.